awx/api/views/__init__.py

```diff
@@ -391,8 +391,10 @@ class InstanceHealthCheck(GenericAPIView):
     permission_classes = (IsSystemAdminOrAuditor,)
 
     def get_queryset(self):
+        obj = self.get_object()
+        if obj.node_type == 'execution':
+            return super().get_queryset()
```

Comment:
If the node type is not execution, this returns `None`, and I don't have any intuition about what happens in that case. Matters for a GET